whylog_vim/gui/windows.py

```diff
@@ -69,7 +69,13 @@ def __init__(self):
         self.windows = dict()
 
     def create_window(self, window_type, content, modifiable=False, size=None):
+        self.origin_filename = VimUIWrapper.get_current_filename()
+        self.origin_offset = VimUIWrapper.get_cursor_offset()
         self.windows[window_type] = Window(window_type, content, modifiable, size)
+        for window in six.iterkeys(self.windows):
+            if self.origin_filename.endswith(window):
```

Comment:
An attribute origin_filename of window is the result of get_current_filename function of VimUIWrapper. It returns the full path pf the file.  For example if the vim was opened in '/home/user/' it will returns '/home/user/whylog_query'. In this line we wants to delete all windows with 'whylog_query' and others suffixes.